tests/app/views/contexts/test_summary_context.py

```diff
@@ -438,5 +438,7 @@ def test_context_for_summary():
         "summary_type": "Summary",
         "view_submitted_response": False,
     }
-
+    print(context)
+    print("---------------------")
```

Comment:
why the print statements?

Reply:
Oh was just using to see context when my debugger was not working 👍